frontend/src/components/ShootWorkers/GWorkerInputGeneric.vue

```diff
@@ -580,6 +580,7 @@ export default {
   }
   .smallInput {
     max-width: 120px;
+    min-width: 112px;
```

Comment:
Why `112px` and not `120px`?

Reply:
112px was the min width for the input fields so that nothing got cut off and without wasting too much space when using 120px. Apart from that there was no other reason

Reply:
Ok. Just wanted to make sure this by intent and not a typo.